plugins/openstack_swift/openstack_swift_stats_.pod

```diff
@@ -0,0 +1,77 @@
+=cut
```

Comment:
Please move this nicely-formatted documentation into the header of the plugin. Otherwise it will not be available for `munin-get` or the [plugin gallery](https://gallery.munin-monitoring.org/).